docs/sources/configuration/_index.md

```diff
@@ -1744,7 +1762,27 @@ memcached_client:
 
   # Whether or not to use a consistent hash to discover multiple memcached servers.
   # CLI flag: -<prefix>.memcached.consistent-hash
-  [consistent_hash: <bool>]
+  [consistent_hash: <bool> | default = true]
```

Comment:
based on the generic placeholders section at the top, boolean values should use the placeholder <boolean>
```suggestion
  [consistent_hash: <boolean> | default = true]
```